pkg/users/authorized-keys-panel.js

```diff
@@ -157,7 +156,7 @@ export function AuthorizedKeys({ name, home, allow_mods }) {
                                                   }}
                                                   toggle={
                                                       <KebabToggle
-                                                      onToggle={open => {
+                                                      onToggle={(_event, open) => {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18672-20230425-151143-725ccf60-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)